pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -128,29 +129,53 @@ func (c Client) fetchAttestationOccurrence(containerImage string, kind string, a
 		return nil, fmt.Errorf("%s is not a valid image hosted in GCR", containerImage)
 	}
 
+	noteName := fmt.Sprintf("%s/notes/%s", auth.Spec.NoteReference, auth.Name)
```

Comment:
NoteReference should IMO contain the entire note name, like in https://cloud.google.com/binary-authorization/docs/reference/rest/v1/projects.attestors.  The name of the note should not be derived from auth.Name.

For example, an AttestationAuthority might look like:
Name: 'authority-qa'
Spec:
  NoteReference: 'projects/some-secure-project/notes/qa-note-123'

Reply:
I agree. Could you please file an issue for this? This definitely fits with the other work you and @ooq are doing for compatibility.

Reply:
Filed https://github.com/grafeas/kritis/issues/420